plugins/inventory/icinga2.py

```diff
@@ -35,13 +35,22 @@
         type: string
         required: true
       host_filter:
-        description: An Icinga2 API valid host filter.
+        description:
+          - An Icinga2 API valid host filter. Leave blank for no filtering
         type: string
         required: false
       validate_certs:
         description: Enables or disables SSL certificate verification.
         type: boolean
         default: true
+      inventory_attr:
+        description:
+          - Allows the override of the inventory name based on different attributes.
+          - This allows for changing the way limits are used.
```

Comment:
How about mentioning that `name` is a better choice than the current default here? (Feel free to add more details to this, I don't know the situation well enough.)
```suggestion
          - This allows for changing the way limits are used.
          - The current default, C(address), is sometimes not unique or present. We recommend to use C(name) instead.
```